<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Toolkit for the automatic analysis of omics data • biokit</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="bootstrap-toc.css">
<script src="bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="pkgdown.css" rel="stylesheet">
<script src="pkgdown.js"></script><meta property="og:title" content="Toolkit for the automatic analysis of omics data">
<meta property="og:description" content="&gt;
    Biokit is an unified wrapper package for functions and utilities to perform 
    the analysis of transcriptomic and proteomic data. It contains several functions
    to carry out a basic exploratory analysis of the omics data tables, together with
    tools to normalize and perform the differential analysis between the sample groups
    of interest. In addition, biokit can also conduct the functional analysis of the 
    results to reduce its dimensionality and increase its interpretability, using 
    over-representation analysis (ORA) or functional class scoring (FCS) approaches. ">
<meta property="og:image" content="https://martingarridorc.github.io/biokit/, https://github.com/martingarridorc/biokit/logo.svg">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-home">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="index.html">biokit</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.1.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="index.html">
    <span class="fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="reference/index.html">Reference</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/martingarridorc/biokit/">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="contents col-md-9">

<div id="biokit-" class="section level1">
<div class="page-header"><h1 class="hasAnchor">
<a href="#biokit-" class="anchor"></a>Biokit <img src="reference/figures/logo.svg" align="right" height="200">
</h1></div>

<p>Biokit is an unified wrapper package for functions and utilities to perform the analysis of transcriptomic and proteomic data. It contains several functions to carry out a basic exploratory analysis of the omics data tables, together with tools to normalize and perform the differential analysis between the sample groups of interest. In addition, biokit can also conduct the functional analysis of the results to reduce its dimensionality and increase its interpretability, using over-representation analysis (ORA) or functional class scoring (FCS) approaches.</p>
<div id="installation" class="section level2">
<h2 class="hasAnchor">
<a href="#installation" class="anchor"></a>Installation</h2>
<p>The package can be installed through the <code>install_github()</code> function from <a href="https://cran.r-project.org/web/packages/devtools/index.html">devtools</a>.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1"></a>devtools<span class="op">::</span><span class="kw">install_github</span>(<span class="dt">repo =</span> <span class="st">"https://github.com/martingarridorc/biokit"</span>)</span></code></pre></div>
</div>
</div>
<div id="case-of-use" class="section level1">
<h1 class="hasAnchor">
<a href="#case-of-use" class="anchor"></a>Case-of-use</h1>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va">biokit</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span><span class="op">(</span><span class="st">"sarsCovData"</span><span class="op">)</span>
<span class="fu"><a href="https://rdrr.io/r/utils/data.html">data</a></span><span class="op">(</span><span class="st">"humanHallmarks"</span><span class="op">)</span>

<span class="co"># set output directory for images</span>
<span class="fu">knitr</span><span class="fu">::</span><span class="va"><a href="https://rdrr.io/pkg/knitr/man/opts_chunk.html">opts_chunk</a></span><span class="op">$</span><span class="fu">set</span><span class="op">(</span>
  fig.path <span class="op">=</span> <span class="st">"man/figures/"</span>
<span class="op">)</span></code></pre></div>
<p>To exemplify the capabilities and features of biokit, we will apply it to a transcriptomic dataset obtained from the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147507">following GEO entry</a>. This dataset contains a subset of the samples analyzed in this project, where the reaction from different human cell lines to SARS-COV-2 infection is evaluated using RNA-Seq. The starting materials for the analysis are:</p>
<p><strong>The RNA-Seq counts table</strong></p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="va">sarsCovMat</span><span class="op">[</span>, <span class="fl">1</span><span class="op">:</span><span class="fl">3</span><span class="op">]</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb4"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1"></a><span class="co">##           NHBE_Mock_1 NHBE_Mock_2 NHBE_Mock_3</span></span>
<span id="cb4-2"><a href="#cb4-2"></a><span class="co">## DDX11L1             0           0           0</span></span>
<span id="cb4-3"><a href="#cb4-3"></a><span class="co">## WASH7P             29          24          23</span></span>
<span id="cb4-4"><a href="#cb4-4"></a><span class="co">## FAM138A             0           0           0</span></span>
<span id="cb4-5"><a href="#cb4-5"></a><span class="co">## FAM138F             0           0           0</span></span>
<span id="cb4-6"><a href="#cb4-6"></a><span class="co">## OR4F5               0           0           0</span></span>
<span id="cb4-7"><a href="#cb4-7"></a><span class="co">## LOC729737         112         119         113</span></span></code></pre></div>
<p><strong>A data frame containing the sample group information</strong></p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="va">sarsCovSampInfo</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb6"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1"></a><span class="co">##                name           group</span></span>
<span id="cb6-2"><a href="#cb6-2"></a><span class="co">## 1       NHBE_Mock_1       NHBE_Mock</span></span>
<span id="cb6-3"><a href="#cb6-3"></a><span class="co">## 2       NHBE_Mock_2       NHBE_Mock</span></span>
<span id="cb6-4"><a href="#cb6-4"></a><span class="co">## 3       NHBE_Mock_3       NHBE_Mock</span></span>
<span id="cb6-5"><a href="#cb6-5"></a><span class="co">## 4 NHBE_SARS.CoV.2_1 NHBE_SARS.CoV.2</span></span>
<span id="cb6-6"><a href="#cb6-6"></a><span class="co">## 5 NHBE_SARS.CoV.2_2 NHBE_SARS.CoV.2</span></span>
<span id="cb6-7"><a href="#cb6-7"></a><span class="co">## 6 NHBE_SARS.CoV.2_3 NHBE_SARS.CoV.2</span></span></code></pre></div>
<p><strong>And a list containing the MSigDb functional categories, that we will use for the functional analysis</strong></p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/base/lapply.html">lapply</a></span><span class="op">(</span><span class="va">humanHallmarks</span><span class="op">[</span><span class="fl">1</span><span class="op">:</span><span class="fl">10</span><span class="op">]</span>, <span class="va">head</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb8"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1"></a><span class="co">## $HALLMARK_ADIPOGENESIS</span></span>
<span id="cb8-2"><a href="#cb8-2"></a><span class="co">## [1] "ABCA1" "ABCB8" "ACAA2" "ACADL" "ACADM" "ACADS"</span></span>
<span id="cb8-3"><a href="#cb8-3"></a><span class="co">## </span></span>
<span id="cb8-4"><a href="#cb8-4"></a><span class="co">## $HALLMARK_ALLOGRAFT_REJECTION</span></span>
<span id="cb8-5"><a href="#cb8-5"></a><span class="co">## [1] "AARS"   "ABCE1"  "ABI1"   "ACHE"   "ACVR2A" "AKT1"  </span></span>
<span id="cb8-6"><a href="#cb8-6"></a><span class="co">## </span></span>
<span id="cb8-7"><a href="#cb8-7"></a><span class="co">## $HALLMARK_ANDROGEN_RESPONSE</span></span>
<span id="cb8-8"><a href="#cb8-8"></a><span class="co">## [1] "ABCC4"   "ABHD2"   "ACSL3"   "ACTN1"   "ADAMTS1" "ADRM1"  </span></span>
<span id="cb8-9"><a href="#cb8-9"></a><span class="co">## </span></span>
<span id="cb8-10"><a href="#cb8-10"></a><span class="co">## $HALLMARK_ANGIOGENESIS</span></span>
<span id="cb8-11"><a href="#cb8-11"></a><span class="co">## [1] "APOH"   "APP"    "CCND2"  "COL3A1" "COL5A2" "CXCL6" </span></span>
<span id="cb8-12"><a href="#cb8-12"></a><span class="co">## </span></span>
<span id="cb8-13"><a href="#cb8-13"></a><span class="co">## $HALLMARK_APICAL_JUNCTION</span></span>
<span id="cb8-14"><a href="#cb8-14"></a><span class="co">## [1] "ACTA1" "ACTB"  "ACTC1" "ACTG1" "ACTG2" "ACTN1"</span></span>
<span id="cb8-15"><a href="#cb8-15"></a><span class="co">## </span></span>
<span id="cb8-16"><a href="#cb8-16"></a><span class="co">## $HALLMARK_APICAL_SURFACE</span></span>
<span id="cb8-17"><a href="#cb8-17"></a><span class="co">## [1] "ADAM10"   "ADIPOR2"  "AFAP1L2"  "AKAP7"    "APP"      "ATP6V0A4"</span></span>
<span id="cb8-18"><a href="#cb8-18"></a><span class="co">## </span></span>
<span id="cb8-19"><a href="#cb8-19"></a><span class="co">## $HALLMARK_APOPTOSIS</span></span>
<span id="cb8-20"><a href="#cb8-20"></a><span class="co">## [1] "ADD1"  "AIFM3" "ANKH"  "ANXA1" "APP"   "ATF3" </span></span>
<span id="cb8-21"><a href="#cb8-21"></a><span class="co">## </span></span>
<span id="cb8-22"><a href="#cb8-22"></a><span class="co">## $HALLMARK_BILE_ACID_METABOLISM</span></span>
<span id="cb8-23"><a href="#cb8-23"></a><span class="co">## [1] "ABCA1" "ABCA2" "ABCA3" "ABCA4" "ABCA5" "ABCA6"</span></span>
<span id="cb8-24"><a href="#cb8-24"></a><span class="co">## </span></span>
<span id="cb8-25"><a href="#cb8-25"></a><span class="co">## $HALLMARK_CHOLESTEROL_HOMEOSTASIS</span></span>
<span id="cb8-26"><a href="#cb8-26"></a><span class="co">## [1] "ABCA2" "ACAT2" "ACSS2" "ACTG1" "ADH4"  "ALCAM"</span></span>
<span id="cb8-27"><a href="#cb8-27"></a><span class="co">## </span></span>
<span id="cb8-28"><a href="#cb8-28"></a><span class="co">## $HALLMARK_COAGULATION</span></span>
<span id="cb8-29"><a href="#cb8-29"></a><span class="co">## [1] "A2M"   "ACOX2" "ADAM9" "ANG"   "ANXA1" "APOA1"</span></span></code></pre></div>
<p>In a first step, we can explore the per-sample value distribution of the raw counts table. Then, we can filter and normalize the count matrix using a minimum cutoff of counts across samples, with a default value of <strong>15</strong>. Then, we can normalize the resulting count matrix with the edgeR TMM approach, using the <code><a href="reference/countsToTmm.html">countsToTmm()</a></code> function from the biokit.</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu">biokit</span><span class="fu">::</span><span class="fu"><a href="reference/violinPlot.html">violinPlot</a></span><span class="op">(</span><span class="va">sarsCovMat</span><span class="op">)</span></code></pre></div>
<p><img src="reference/figures/unnamed-chunk-5-1.png"><!-- --></p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">sarsCovMat</span> <span class="op">&lt;-</span> <span class="va">sarsCovMat</span><span class="op">[</span><span class="fu"><a href="https://rdrr.io/r/base/colSums.html">rowSums</a></span><span class="op">(</span><span class="va">sarsCovMat</span><span class="op">)</span> <span class="op">&gt;=</span> <span class="fl">15</span>, <span class="op">]</span>
<span class="va">tmmMat</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/countsToTmm.html">countsToTmm</a></span><span class="op">(</span><span class="va">sarsCovMat</span><span class="op">)</span></code></pre></div>
<p>Next, we can explore the new per-sample value distribution using again the violin plot function.</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu">biokit</span><span class="fu">::</span><span class="fu"><a href="reference/violinPlot.html">violinPlot</a></span><span class="op">(</span><span class="va">tmmMat</span><span class="op">)</span></code></pre></div>
<p><img src="reference/figures/unnamed-chunk-6-1.png"><!-- --></p>
<p>In a second exploratory step, we can apply a Principal Component Analysis (PCA) to reduce the dataset dimensionality and explore the group distribution in a bidimensional space formed by the first two principal components.</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu">biokit</span><span class="fu">::</span><span class="fu"><a href="reference/pcaPlot.html">pcaPlot</a></span><span class="op">(</span>mat <span class="op">=</span> <span class="va">tmmMat</span>, sampInfo <span class="op">=</span> <span class="va">sarsCovSampInfo</span>, groupCol <span class="op">=</span> <span class="st">"group"</span><span class="op">)</span></code></pre></div>
<p><img src="reference/figures/unnamed-chunk-7-1.png"><!-- --></p>
<p>Next, we can explore the top 25 genes with the higuest standard deviation in teh entire dataset, representing and clustering them through a heatmap representation.</p>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="reference/heatmapPlot.html">heatmapPlot</a></span><span class="op">(</span>mat <span class="op">=</span> <span class="va">tmmMat</span>, sampInfo <span class="op">=</span> <span class="va">sarsCovSampInfo</span>, groupCol <span class="op">=</span> <span class="st">"group"</span>,scaleBy <span class="op">=</span> <span class="st">"row"</span>,  nTop <span class="op">=</span> <span class="fl">25</span><span class="op">)</span></code></pre></div>
<p><img src="reference/figures/unnamed-chunk-8-1.png"><!-- --></p>
<p>Once that we have evaluated the distribution of sample groups and of most variable genes with basic exploratory analysis, we can perform a differential expression between the sample groups of interest using the for the linear models included in the limma package. The <code><a href="reference/volcanoPlot.html">volcanoPlot()</a></code> function can be used to obtain a broad spectrum view of the results for each of the comparisons carried out.</p>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">diffRes</span> <span class="op">&lt;-</span> <span class="fu">biokit</span><span class="fu">::</span><span class="fu"><a href="reference/autoLimmaComparison.html">autoLimmaComparison</a></span><span class="op">(</span>mat <span class="op">=</span> <span class="va">tmmMat</span>, sampInfo <span class="op">=</span> <span class="va">sarsCovSampInfo</span>, groupCol <span class="op">=</span> <span class="st">"group"</span><span class="op">)</span>
<span class="fu">biokit</span><span class="fu">::</span><span class="fu"><a href="reference/volcanoPlot.html">volcanoPlot</a></span><span class="op">(</span><span class="va">diffRes</span><span class="op">)</span></code></pre></div>
<p><img src="reference/figures/unnamed-chunk-9-1.png"><!-- --></p>
<p>In a final step, we can perform the functional analysis for each comparison using the GSEA approach and visualize the significant results using the <code><a href="reference/gseaPlot.html">gseaPlot()</a></code> function.</p>
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">gseaResults</span> <span class="op">&lt;-</span> <span class="fu"><a href="reference/gseaFromStats.html">gseaFromStats</a></span><span class="op">(</span>df <span class="op">=</span> <span class="va">diffRes</span>, funCatList <span class="op">=</span> <span class="va">humanHallmarks</span>, rankCol <span class="op">=</span> <span class="st">"logFc"</span>, splitCol <span class="op">=</span> <span class="st">"comparison"</span><span class="op">)</span>
<span class="fu"><a href="reference/gseaPlot.html">gseaPlot</a></span><span class="op">(</span><span class="va">gseaResults</span><span class="op">)</span></code></pre></div>
<p><img src="reference/figures/unnamed-chunk-10-1.png"><!-- --></p>
<p>Session information</p>
<div class="sourceCode" id="cb16"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html">sessionInfo</a></span><span class="op">(</span><span class="op">)</span></code></pre></div>
<div class="sourceCode" id="cb17"><pre class="sourceCode R"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1"></a><span class="co">## R version 4.0.5 (2021-03-31)</span></span>
<span id="cb17-2"><a href="#cb17-2"></a><span class="co">## Platform: x86_64-w64-mingw32/x64 (64-bit)</span></span>
<span id="cb17-3"><a href="#cb17-3"></a><span class="co">## Running under: Windows 10 x64 (build 19042)</span></span>
<span id="cb17-4"><a href="#cb17-4"></a><span class="co">## </span></span>
<span id="cb17-5"><a href="#cb17-5"></a><span class="co">## Matrix products: default</span></span>
<span id="cb17-6"><a href="#cb17-6"></a><span class="co">## </span></span>
<span id="cb17-7"><a href="#cb17-7"></a><span class="co">## locale:</span></span>
<span id="cb17-8"><a href="#cb17-8"></a><span class="co">## [1] LC_COLLATE=English_United Kingdom.1252 </span></span>
<span id="cb17-9"><a href="#cb17-9"></a><span class="co">## [2] LC_CTYPE=English_United Kingdom.1252   </span></span>
<span id="cb17-10"><a href="#cb17-10"></a><span class="co">## [3] LC_MONETARY=English_United Kingdom.1252</span></span>
<span id="cb17-11"><a href="#cb17-11"></a><span class="co">## [4] LC_NUMERIC=C                           </span></span>
<span id="cb17-12"><a href="#cb17-12"></a><span class="co">## [5] LC_TIME=English_United Kingdom.1252    </span></span>
<span id="cb17-13"><a href="#cb17-13"></a><span class="co">## </span></span>
<span id="cb17-14"><a href="#cb17-14"></a><span class="co">## attached base packages:</span></span>
<span id="cb17-15"><a href="#cb17-15"></a><span class="co">## [1] stats     graphics  grDevices utils     datasets  methods   base     </span></span>
<span id="cb17-16"><a href="#cb17-16"></a><span class="co">## </span></span>
<span id="cb17-17"><a href="#cb17-17"></a><span class="co">## other attached packages:</span></span>
<span id="cb17-18"><a href="#cb17-18"></a><span class="co">## [1] biokit_0.1.1</span></span>
<span id="cb17-19"><a href="#cb17-19"></a><span class="co">## </span></span>
<span id="cb17-20"><a href="#cb17-20"></a><span class="co">## loaded via a namespace (and not attached):</span></span>
<span id="cb17-21"><a href="#cb17-21"></a><span class="co">##  [1] locfit_1.5-9.4              Rcpp_1.0.6                 </span></span>
<span id="cb17-22"><a href="#cb17-22"></a><span class="co">##  [3] lattice_0.20-41             tidyr_1.1.3                </span></span>
<span id="cb17-23"><a href="#cb17-23"></a><span class="co">##  [5] assertthat_0.2.1            digest_0.6.27              </span></span>
<span id="cb17-24"><a href="#cb17-24"></a><span class="co">##  [7] utf8_1.2.1                  R6_2.5.0                   </span></span>
<span id="cb17-25"><a href="#cb17-25"></a><span class="co">##  [9] GenomeInfoDb_1.26.4         stats4_4.0.5               </span></span>
<span id="cb17-26"><a href="#cb17-26"></a><span class="co">## [11] RSQLite_2.2.5               evaluate_0.14              </span></span>
<span id="cb17-27"><a href="#cb17-27"></a><span class="co">## [13] highr_0.8                   ggplot2_3.3.3              </span></span>
<span id="cb17-28"><a href="#cb17-28"></a><span class="co">## [15] pillar_1.6.1                zlibbioc_1.36.0            </span></span>
<span id="cb17-29"><a href="#cb17-29"></a><span class="co">## [17] rlang_0.4.10                data.table_1.14.0          </span></span>
<span id="cb17-30"><a href="#cb17-30"></a><span class="co">## [19] blob_1.2.1                  S4Vectors_0.28.1           </span></span>
<span id="cb17-31"><a href="#cb17-31"></a><span class="co">## [21] Matrix_1.3-2                rmarkdown_2.7              </span></span>
<span id="cb17-32"><a href="#cb17-32"></a><span class="co">## [23] labeling_0.4.2              BiocParallel_1.24.1        </span></span>
<span id="cb17-33"><a href="#cb17-33"></a><span class="co">## [25] stringr_1.4.0               pheatmap_1.0.12            </span></span>
<span id="cb17-34"><a href="#cb17-34"></a><span class="co">## [27] RCurl_1.98-1.3              bit_4.0.4                  </span></span>
<span id="cb17-35"><a href="#cb17-35"></a><span class="co">## [29] munsell_0.5.0               fgsea_1.16.0               </span></span>
<span id="cb17-36"><a href="#cb17-36"></a><span class="co">## [31] DelayedArray_0.16.3         compiler_4.0.5             </span></span>
<span id="cb17-37"><a href="#cb17-37"></a><span class="co">## [33] xfun_0.22                   pkgconfig_2.0.3            </span></span>
<span id="cb17-38"><a href="#cb17-38"></a><span class="co">## [35] BiocGenerics_0.36.0         htmltools_0.5.1.1          </span></span>
<span id="cb17-39"><a href="#cb17-39"></a><span class="co">## [37] tidyselect_1.1.0            SummarizedExperiment_1.20.0</span></span>
<span id="cb17-40"><a href="#cb17-40"></a><span class="co">## [39] tibble_3.1.0                gridExtra_2.3              </span></span>
<span id="cb17-41"><a href="#cb17-41"></a><span class="co">## [41] GenomeInfoDbData_1.2.4      edgeR_3.32.1               </span></span>
<span id="cb17-42"><a href="#cb17-42"></a><span class="co">## [43] IRanges_2.24.1              matrixStats_0.58.0         </span></span>
<span id="cb17-43"><a href="#cb17-43"></a><span class="co">## [45] fansi_0.4.2                 crayon_1.4.1               </span></span>
<span id="cb17-44"><a href="#cb17-44"></a><span class="co">## [47] dplyr_1.0.6                 bitops_1.0-6               </span></span>
<span id="cb17-45"><a href="#cb17-45"></a><span class="co">## [49] grid_4.0.5                  gtable_0.3.0               </span></span>
<span id="cb17-46"><a href="#cb17-46"></a><span class="co">## [51] lifecycle_1.0.0             DBI_1.1.1                  </span></span>
<span id="cb17-47"><a href="#cb17-47"></a><span class="co">## [53] magrittr_2.0.1              scales_1.1.1               </span></span>
<span id="cb17-48"><a href="#cb17-48"></a><span class="co">## [55] stringi_1.5.3               cachem_1.0.4               </span></span>
<span id="cb17-49"><a href="#cb17-49"></a><span class="co">## [57] farver_2.1.0                XVector_0.30.0             </span></span>
<span id="cb17-50"><a href="#cb17-50"></a><span class="co">## [59] limma_3.46.0                ggfortify_0.4.11           </span></span>
<span id="cb17-51"><a href="#cb17-51"></a><span class="co">## [61] ellipsis_0.3.2              generics_0.1.0             </span></span>
<span id="cb17-52"><a href="#cb17-52"></a><span class="co">## [63] vctrs_0.3.8                 fastmatch_1.1-0            </span></span>
<span id="cb17-53"><a href="#cb17-53"></a><span class="co">## [65] RColorBrewer_1.1-2          tools_4.0.5                </span></span>
<span id="cb17-54"><a href="#cb17-54"></a><span class="co">## [67] bit64_4.0.5                 Biobase_2.50.0             </span></span>
<span id="cb17-55"><a href="#cb17-55"></a><span class="co">## [69] glue_1.4.2                  purrr_0.3.4                </span></span>
<span id="cb17-56"><a href="#cb17-56"></a><span class="co">## [71] MatrixGenerics_1.2.1        parallel_4.0.5             </span></span>
<span id="cb17-57"><a href="#cb17-57"></a><span class="co">## [73] fastmap_1.1.0               yaml_2.2.1                 </span></span>
<span id="cb17-58"><a href="#cb17-58"></a><span class="co">## [75] AnnotationDbi_1.52.0        colorspace_2.0-0           </span></span>
<span id="cb17-59"><a href="#cb17-59"></a><span class="co">## [77] GenomicRanges_1.42.0        memoise_2.0.0              </span></span>
<span id="cb17-60"><a href="#cb17-60"></a><span class="co">## [79] knitr_1.31</span></span></code></pre></div>
</div>

  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <div class="links">
<h2>Links</h2>
<ul class="list-unstyled">
<li>Browse source code at <br><a href="https://github.com/martingarridorc/biokit/">https://​github.com/​martingarridorc/​biokit/​</a>
</li>
</ul>
</div>
<div class="license">
<h2>License</h2>
<ul class="list-unstyled">
<li>
<a href="https://www.r-project.org/Licenses/GPL-3">GPL-3</a> + file <a href="LICENSE-text.html">LICENSE</a>
</li>
</ul>
</div>
<div class="developers">
<h2>Developers</h2>
<ul class="list-unstyled">
<li>Martín Garrido Rodríguez-Córdoba <br><small class="roles"> Author, maintainer </small> <a href="https://orcid.org/0000-0003-4125-5643" target="orcid.widget" aria-label="ORCID"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
</ul>
</div>

  <div class="dev-status">
<h2>Dev status</h2>
<ul class="list-unstyled">
<li><a href="https://travis-ci.com/martingarridorc/biokit"><img src="https://travis-ci.com/martingarridorc/biokit.svg?branch=master" alt="Build Status"></a></li>
<li><a href="https://codecov.io/gh/martingarridorc/biokit/"><img src="https://codecov.io/gh/martingarridorc/biokit/branch/master/graph/badge.svg" alt="CodeCov"></a></li>
<li><a href="https://github.com/martingarridorc/biokit/blob/master/LICENSE"><img src="https://img.shields.io/github/license/martingarridorc/biokit.svg?color=yellow" alt="License"></a></li>
</ul>
</div>
</div>
</div>


      <footer><div class="copyright">
  <p>Developed by Martín Garrido Rodríguez-Córdoba.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
